Doc/whatsnew/3.12.rst

```diff
@@ -188,7 +191,6 @@ although there is currently no date scheduled for their removal.
   :keyword:`for`, :keyword:`if`, :keyword:`in`, :keyword:`is` and :keyword:`or`.
   In a future release it will be changed to a syntax error. (:gh:`87999`)
 
-
```

Comment:
I'm sorry. I tried removing the extra blank line but removed one line extra. Could you please add back a newline?